code/run_templates/filter1/etl.py

```diff
@@ -0,0 +1,92 @@
+import shutil
+import os
+import sys
+import cosmotech_api
+from cosmotech_api.model.dataset import Dataset
+from pathlib import Path
+
+sys.path.append(str(Path(__file__).parents[1] / "etl_sub_dataset_by_filter"))
+
+from twingraph import (
+    parse_twingraph_json,
+    create_csv_files_from_graph_content,
+    upload_twingraph_zip_archive,
+)
+from common import get_logger, get_api
+
+LOGGER = get_logger()
+
+
+def main():
+    LOGGER.info("Starting the ETL Run")
+    organization_id = os.environ.get("CSM_ORGANIZATION_ID")
+    workspace_id = os.environ.get("CSM_WORKSPACE_ID")
+    runner_id = os.environ.get("CSM_RUNNER_ID")
+    api = get_api()
+
+    runner = api["runner"].get_runner(
+        organization_id=organization_id, workspace_id=workspace_id, runner_id=runner_id
+    )
+    subdataset_id = runner["dataset_list"][0]
+    parent_dataset_id = runner["dataset_list"][1]
+    graph_filter = {
+        "key": "Thirsty",
+        "value": (runner["parameters_values"][0]["value"] == "THIRSTY"),
```

Comment:
This might not be a good practice to access parameters values by index, because the webapp and/or back-end may send them in a random order (we only have one in our case though, so it will work anyway :slightly_smiling_face: )